### ISSUE TITLE
Merging multiple large potree files after conversion

### ISSUE THREAD

**Issue by maramusag** [potree/potree#1107]
We have many massive las file which we are converting into potree file with potreeconverter.exe

This obviously takes  a long time.
We also got this one error in the distributing after the counting was complete
![image](https://user-images.githubusercontent.com/89246627/141737772-b430c897-4a51-4585-a53c-5ee59e377ecb.png)

Is there a way to merge multiple potree files? 
This was we would convert individual las into potree, and then start merging our potree files

**Comment by HeadLessHUN** [comment]
you cannot merge them. However you could load multiple cloud to potree viewer

**Comment by maramusag** [comment]
> you cannot merge them. However you could load multiple cloud to potree viewer

yes you can load multiple clouds to the potree viewer...but we have 1000s of large scans....it takes forever to load up in the viewer.
However one large global potree cloud loads lovely and fast...

**Comment by HeadLessHUN** [comment]
you could create fewer bigger cloud for example like 10 100setup or similar, but after conversion there is no way before conversion yes, you can list as many file to the converter as you want. or just place them in a directory and give potree that directory... You don't need to merge them before conversion, that way maybe it would handle the clouds differently and wouldn't give an error.

**Comment by maramusag** [comment]
Yes, we have considered all this, but when potree has multiple clouds it slows down significantly when loading in the viewer(even a fewer bigger clouds).
It only loads fast with one large potree cloud file.
So we are stuck, how to load fast with multiple potree file?(which is slow) 
or have one large potree file(but means combine many las, and can error)
Thus, we want to merge many potree files.

Would love to hear any other suggestions

**Comment by HeadLessHUN** [comment]
in the old converter version there was a way to append potree octrees, you just have to calculate/set the bounding boy beforehand, i'don't know if this is still inlcuded in 2.0 but i don't really think.

the other thing which uses the old/outdated potree converter, is this repository [https://github.com/NLeSC/Massive-PotreeConverter](https://github.com/NLeSC/Massive-PotreeConverter) it does exactly what you want, butI've never tried that.

**Comment by maramusag** [comment]
> in the old converter version there was a way to append potree octrees, you just have to calculate/set the bounding boy beforehand, i'don't know if this is still inlcuded in 2.0 but i don't really think.
> 
> the other thing which uses the old/outdated potree converter, is this repository https://github.com/NLeSC/Massive-PotreeConverter it does exactly what you want, butI've never tried that.

thanks so much, this looks like exactly what we need. Will post the results

**Comment by Janphr** [comment]
I'm trying to do exactly the same thing. Have you found a solution?
